geopandas/base.py

```diff
@@ -11,7 +11,7 @@
 import geopandas as gpd
 
 from .array import GeometryArray, GeometryDtype
-from .sindex import get_sindex_class, has_sindex
+from .sindex import has_sindex
 
 # for backwards compat
 # this will be static (will NOT follow USE_PYGEOS changes)
```

Comment:
Do we want to remove this (`base.HAS_SINDEX`) attribute at some point, now that we have the `sindex.has_sindex()` method?

Reply:
I guess so, but not within this PR.

Reply:
Will open a follow-up issue for this. It's probably worth getting rid of. I'm just not sure how we would issue a warning...